app/views/layouts/application.html.erb

```diff
@@ -1,22 +1,10 @@
 <!DOCTYPE html>
 <html>
-  <header class="business-header">
-        <div class="container">
-            <div class="row">
-                <div class="col-lg-12  text-center">
-                    <br><br><br><br><br><br>
-                    <hr>
-                    <a href="/">
-                      <h1 class="tagline">WeEatAgain</h1>
-                    </a>
-                    <hr>
-                    <p>Is is lunchtime already?</p>
-                    <p>Choose a place to eat out of these recommended resturants</p>
-                    <br>
-                </div>
-            </div>
-        </div>
-  </header>
+<script src="https://npmcdn.com/tether@1.2.4/dist/js/tether.min.js"></script>
+
+<script src="https://npmcdn.com/bootstrap@4.0.0-alpha.5/dist/js/bootstrap.min.js"></script>
+<%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= react_component("Header") %>
```

Comment:
I think `<header>` would be better in the `<body>`, easier to separate the `<head>` from the content.